staking_deposit/cli/generate_keys.py

```diff
@@ -105,6 +105,12 @@ def generate_keys_arguments_decorator(function: Callable[..., Any]) -> Callable[
             param_decls=['--execution_address', '--eth1_withdrawal_address'],
             prompt=lambda: load_text(['arg_execution_address', 'prompt'], func='generate_keys_arguments_decorator'),
         ),
+        jit_option(
+            default=False,
+            is_flag=True,
+            param_decls='--pbkdf2',
+            help='Uses the pbkdf2 encryption method instead of scrypt.',
```

Comment:
This should be moved into the internalization structure with the `load_text` function. The message should also be explicit about this being used for the resulting keystore file(s).